### DISCUSSION TITLE
Using environment variables in theme

### DISCUSSION THREAD

**Discussion by samEngineering** [aristocratos/btop#289] [Ideas]
Greetings ^_^
Is anyone aware of a way to use environment variables in a custom theme file?
I'm trying to do something like this:
```
theme[cpu_box]="${GREEN}"
```
If this functionality is not available, this would be a great feature :)  btop++ ftw!
Thanks in advance.

**Comment by wimstefan** [comment]
That would be great!
Another idea is to get the colours via xrdb [similar to polybar](https://github.com/polybar/polybar/wiki/Configuration#referencing-values) ...
